src/camera.js

```diff
@@ -42,6 +43,8 @@ vgl.camera = function () {
       m_right = 1.0,
       m_top = +1.0,
       m_bottom = -1.0,
+      m_parallelExtents = [undefined, undefined, 1, 256],
+      m_unitsPerPixel,
```

Comment:
No.  Also, I'll refactor the parallelExtents to be a dictionary so that everything is clearer.